tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -111,5 +116,22 @@ func expectStatus(t *testing.T, ctx resource.Context, ns namespace.Instance, has
 	if strings.Contains(status, msg.ReferencedResourceNotFound.Code()) != hasError {
 		return fmt.Errorf("expected error=%v, but got %v", hasError, status)
 	}
+	conditions, ok := x.Object["status"].(map[string]interface{})["conditions"]
+	if !ok {
+		return fmt.Errorf("expected conditions to exist, but got %v", status)
+	}
+	found := false
+	for _, ucondition := range conditions.([]interface{}) {
+		condition := ucondition.(map[string]interface{})
```

Comment:
a typed struct would be great

Reply:
Agreed.  I think I have a function for that somewhere... A simple cast doesn't work unfortunately...